### ISSUE TITLE
"rust-analyzer.server.path" doesn't work in MacOS

### ISSUE THREAD

**Issue by wisonye** [fannheyward/coc-rust-analyzer#643]
It seems `rust-analyzer.server.path` doesn't work on the latest version on Mac (it works fine in Linux, or maybe I didn't upgrade it in Linux).

I got the settings below in coc config, installed via `brew install rust-analyzer` before:

```json
"rust-analyzer.server.path": "/usr/local/Cellar/rust-analyzer/2021-02-15/bin/rust-analyzer",
```

But it reports error like below:

<img width="1356" alt="custom-version-not-working" src="https://user-images.githubusercontent.com/3477321/109761032-a3ac3080-7c54-11eb-859b-1c5f815d2989.png">

When I open `Rust` project, the plugin doesn't detect that version and ask me to download the latest one. I confirmed that it exists:

<img width="990" alt="Screen Shot 2021-03-03 at 7 15 21 PM" src="https://user-images.githubusercontent.com/3477321/109761259-f1c13400-7c54-11eb-94d1-29d58469c9b9.png">

**Comment by fannheyward** [comment]
What's your coc version and coc-rust-analyzer version?

**Comment by wisonye** [comment]
> What's your coc version and coc-rust-analyzer version?

```js
## versions

vim version: NVIM v0.4.4
node version: v14.16.0
coc.nvim version: 0.0.80-6e5a2aaeb5
coc.nvim directory: /Users/wison/.config/nvim/plugged/coc.nvim
term: xterm-256color
platform: darwin

## Log of coc.nvim

2021-03-03T20:46:52.726 INFO (pid:604) [plugin] - coc.nvim 0.0.80-6e5a2aaeb5 initialized with node: v14.16.0 after 143ms
2021-03-03T20:46:54.997 INFO (pid:604) [services] - registered service "rust-analyzer"
2021-03-03T20:46:54.998 INFO (pid:604) [services] - Rust Analyzer Language Server state change: stopped => starting
2021-03-03T20:46:55.005 INFO (pid:604) [language-client-index] - Language server "rust-analyzer" started with 625
2021-03-03T20:46:55.029 INFO (pid:604) [services] - Rust Analyzer Language Server state change: starting => running
2021-03-03T20:46:55.036 INFO (pid:604) [services] - service rust-analyzer started
2021-03-03T20:47:01.726 INFO (pid:604) [attach] - receive notification: showInfo []
```

<img width="779" alt="Screen Shot 2021-03-03 at 8 48 45 PM" src="https://user-images.githubusercontent.com/3477321/109771624-e6c0d080-7c61-11eb-9aac-a7a70a4aba14.png">

**Comment by fannheyward** [comment]
Your coc-rust-analyzer is very old, the latest release is v0.36.0, please upgrade it by `:CocUpdate`.

**Comment by wisonye** [comment]
> Your coc-rust-analyzer is very old, the latest release is v0.36.0, please upgrade it by `:CocUpdate`.

Thanks for pointing it out, update fixed it. Sorry about that, I will double-check next time before I create the new issue:)
